### DISCUSSION TITLE
release: v0.0.25 (#333)

### DISCUSSION THREAD

**Discussion by AzulGarza** [TimeCopilot/timecopilot#334] [General]
### Features

* **TimeGPT finetuning**: Finetuning is now supported for TimeGPT. You can adapt the pre-trained model to your data before forecasting via `TimeGPTFinetuningConfig`, with options for loss function and finetuning depth. See [#332](https://github.com/TimeCopilot/timecopilot/pull/332) and the [Finetuning Foundation Models](https://timecopilot.dev/examples/finetuning/) example for a full walkthrough.

    ```python
    import pandas as pd
    from timecopilot.models.foundation.timegpt import TimeGPT, TimeGPTFinetuningConfig

    df = pd.read_csv(
        "https://timecopilot.s3.amazonaws.com/public/data/events_pageviews.csv",
        parse_dates=["ds"],
    )

    model = TimeGPT(
        finetuning_config=TimeGPTFinetuningConfig(
            finetune_steps=10,
            finetune_loss="mse",
        ),
        alias="TimeGPT-finetuned",
    )
    ```

### Documentation

* **Finetuning evaluation example**: Added a finetuning evaluation section to the [Finetuning Foundation Models](https://timecopilot.dev/examples/finetuning/) notebook, comparing MAPE across different `finetune_steps` values via cross-validation. See [#326](https://github.com/TimeCopilot/timecopilot/pull/326).

---

**Full Changelog**: https://github.com/TimeCopilot/timecopilot/compare/v0.0.24...v0.0.25

<hr /><em>This discussion was created from the release <a href='https://github.com/TimeCopilot/timecopilot/releases/tag/v0.0.25'>release: v0.0.25 (#333)</a>.</em>
